libpod/container_internal.go

```diff
@@ -758,7 +758,7 @@ func (c *Container) isStopped() (bool, error) {
 		return true, err
 	}
 
-	return !c.ensureState(define.ContainerStateRunning, define.ContainerStatePaused), nil
+	return !c.ensureState(define.ContainerStateRunning, define.ContainerStatePaused, define.ContainerStateStopping), nil
```

Comment:
Note to self: in future we should require all of these `ensureState` invocations to not be inverted - will make them safer when we add states, as all valid states will have to be explicitly listed.

Reply:
+1

Reply:
Or add private functions for `isRunning` etc.